lib/pf/inline/accounting.pm

```diff
@@ -211,7 +214,10 @@ sub inline_accounting_maintenance {
         # Commit database transaction
         unless ($dbh->commit) {
             $logger->error("Error while committing database transaction: " . $dbh->errstr);
-            $dbh->rollback or $logger->logdie("Can't rollback database transaction: " . $dbh->errstr);
+            unless($dbh->rollback){
+                $logger->error("Can't rollback database transaction: " . $dbh->errstr);
+                return undef;
+            }
         }
     }
```

Comment:
Check to see if additional error handling is needed where inline_accounting_maintenance is called